lib/cli.coffee

```diff
@@ -25,12 +26,20 @@ installMycha = (testHelperPath, done) ->
   installer.install done
 
 
+loadConfig = (done) ->
```

Comment:
Agreed that the file should be renamed to a verb if you go with the simple function export. I don't feel strongly about either way.